apps/prairielearn/src/lib/externalGraderLocal.ts

```diff
@@ -73,12 +74,19 @@ export class ExternalGraderLocal {
 
       if (question.external_grading_entrypoint?.includes('serverFilesCourse')) {
         // Mark the entrypoint as executable if it lives in serverFilesCourse.
-        // If it is living in the docker container then we don't have access to it before
-        // we actually run it.
+        // If it is living in the docker container then we don't have access to
+        // it before we actually run it.
+        //
+        // TODO: This code differs from grader-host, in that it only runs for
+        // serverFilesCourse, while grader-host runs for all entrypoints. There
+        // should be consistency in this case to address cases where the
+        // entrypoint is, e.g., in /grade/tests instead. It also assumes that
+        // `serverFilesCourse` is prefixed by `/grade/`, which is a reasonable
+        // assumption, but should probably be checked.
```

Comment:
@nwalters512 similar question, but in this case this should replace the `serverFilesCourse` check with a `/grade` test instead. Can you again double check?

Reply:
This sounds about right, yeah. I think this was specifically trying to handle `/grade/serverFilesCourse`, but it seems sensible to generalize this to `/grade/...`.

Reply:
Which seems like it was a problem in the past...
https://github.com/PrairieLearn/PrairieLearn/blob/c74134815f90bbc8144489d44c28b44a95f46541/docs/c-grader/index.md#L26-L31

Reply:
Done in 4b6bccca9dc87cba8c664ff70211fff739e0e3b0.